skaffold.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: skaffold/v1alpha2
+kind: Config
+build:
+
+  artifacts:
+  - imageName: errordeveloper/podinfo
```

Comment:
Can we avoid having the repo name in the image path? For local Kubernetes could it work without `errordeveloper/`?

Reply:
Done!